### DISCUSSION TITLE
See structure while writing query

### DISCUSSION THREAD

**Discussion by mamakawama** [Sequel-Ace/Sequel-Ace#1255] [Ideas]
As a user, I would like to be able to see the table structure while writing a query.

Currently I get the Table Information area below the list of tables. This data is mostly uninteresting when I'm writing queries. In its place the selected table structure would make writing my queries easier.

**Comment by pablomedok** [comment]
It would be also great to be able to drag table items from the list to the query editor to place their names into query and to drag fields from the structure suggested by @mamakawama to place them in to the query, too. 

Another way of solving the problem would be expanding tables hierarchically as folders in Finder's table view to reveal field names.

It would be great to see data types icons in the list before field names to understand the get syntax right: ab String, # Number, ⦿ Boolean, ⌚︎ Date/time, ▧ Memo/Blob, etc.
